docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -91,42 +93,36 @@ In our example, we will be using `java-spring-boot` to deploy a sample https://s
 
 == Deploying a Java Spring Boot® component to an OpenShift cluster
 
-In this example we will be deploying an https://github.com/odo-devfiles/springboot-ex[example Spring Boot® component] that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
+In this example we will be deploying a sample project associated with the java-spring-boot devfile that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
 
-. Download the example Spring Boot® component
+. Create a directory to store the source code for the component:
 +
 [source,sh]
 ----
- $ git clone https://github.com/odo-devfiles/springboot-ex
+ $ mkdir <directory-name>
 ----
 
-. Change the current directory to the component directory:
+. Create a component configuration of Spring Boot component-type named myspring and download its sample project:
```

Comment:
I think the sentence is a bit too long. Perhaps we can split this into two sentences / two steps?

Also forgetting ® in Spring Boot

Reply:
Reverted to the old sentence since I took out `--downloadSource`